docs/changes.rst

```diff
@@ -3,8 +3,13 @@
 Changelog
 =========
 
-`1.2`_ - TBA
-------------
+`Unreleased`_
```

Comment:
```suggestion
`1.2`_ - TBA
```
This needs to be revered, 1.2 is already a known link, I think the other one will break the docs build.